README.md

```diff
@@ -21,32 +39,33 @@ $ psql -f db/schema.sql -d soapbox_dev
 
 2. **Build and install the Soapbox API server (soapboxd) and CLI client (soapboxcli):**
 ``` shell
+$ mkdir -p $GOPATH/src/github.com/adhocteam
```

Comment:
GOPATH is not required to be set as of Go 1.8. It defaults to $HOME/go. This command would fail if GOPATH is not set. To be safe you can replace $GOPATH with $(go env GOPATH)